pyproject.toml

```diff
@@ -34,6 +33,7 @@ alembic = "^1.13.2"
 psycopg2-binary = "^2.9.9"
 apprise = "^1.8.1"
 subliminal = "^2.2.1"
+rank-torrent-name = {path = "/home/spoked/projects/rtn"}
```

Comment:
cough cough 😛

Reply:
this branch was just a placeholder for the changes lol, thanks :D